src/VolleyManagement.Backend/VolleyManagement.Domain/PlayersAggregate/Player.cs

```diff
@@ -36,7 +36,7 @@ public string FirstName
             {
                 if (PlayerValidation.ValidateFirstName(value))
                 {
-                    throw new ArgumentException(Resources.ValidationPlayerFirstName, "FirstName");
+                    throw new ArgumentException(Resources.ValidationPlayerFirstName, $"FirstName");
```

Comment:
What is the point of this change?

Reply:
Sonar warn on in first

Reply:
"Parameter names used into ArgumentException constructors should match an existing one"

And I didn't find another way how to fix it correctly.

Reply:
So how I have to fix it?

Reply:
use nameof(Player.FirstName)

Reply:
![image](https://user-images.githubusercontent.com/28676005/36483895-1c00949c-1720-11e8-9b5b-7fe9f0a2419b.png)